ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -169,3 +169,51 @@ def verification_ui(self):
         for err in self.err_list:
             logger.error(err)
         assert len(self.err_list) == 0, f"{self.err_list}"
+
+    def check_capacity_breakdown(self, project_name, pod_name):
+        """
+        Check Capacity Breakdown
+
+        Args:
+            project_name (str): The name of the project
+            pod_name (str): The name of pod
+
+        Returns:
+            bool: True if project_name and pod_name exist on capacity_breakdown, False otherwise
+
+        """
+        self.navigate_overview_page()
```

Comment:
In OCP and ODF 4.9, the navigation to Overview no longer exists and the capacity breakdown chart is available under Storage-ODF-StorageSystems-ocs-storagecluster-storagesystem.
Also, the default navigation is taken to Block and File tab. Make sure to add conditions if we are verifying something on Object tab as well.

Reply:
I did not automate it on OCP4.9

Reply:
This PR should also be compatible with OCP and ODF 4.9. Please add those steps and run PR validation job.